client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -482,24 +482,32 @@ function returnModalCopyForProduct(
 	const productType =
 		isPlan( product ) && hasDomainsInCart ? 'plan with dependencies' : product.product_slug;
 	const isRenewal = isWpComProductRenewal( product );
-	return returnModalCopy( productType, translate, createUserAndSiteBeforeTransaction, isPwpoUser, isRenewal );
+	return returnModalCopy(
+		productType,
+		translate,
+		createUserAndSiteBeforeTransaction,
+		isPwpoUser,
+		isRenewal
+	);
 }
 
 function returnModalCopy(
 	productType: string,
 	translate: ReturnType< typeof useTranslate >,
 	createUserAndSiteBeforeTransaction: boolean,
 	isPwpoUser: boolean,
-	isRenewal: boolean = false,
+	isRenewal = false
```

Comment:
Just out of curiosity, what is the benefit of removing the type here?

Reply:
The linter complained? 🙃

Reply:
Ah, was it `Type boolean trivially inferred from a boolean literal, remove type annotation. (@typescript-eslint/no-inferrable-types)`? That's interesting! I guess that TS assumes that if your default value is `false`, then the type must be a boolean, although that seems risky to me since the type could be something weird like `false|string`. Still, I trust the compiler here; if it already assumes that's a boolean, then it should guard its use, so this is probably fine.

Reply:
That's the one! When I added the value, I think my environment may not have been fully up to date.

I do agree that I'd like the type to be explicit, but I am not going to get into an argument with the linter!